zstacklib/zstacklib/utils/sanlock.py

```
@@ -374,26 +491,23 @@ def direct_dump_resource(path, offset, size=SMALL_ALIGN_SIZE):
     return bash.bash_roe("sanlock direct dump %s:%s:%s" % (path, offset, size))
 
 @bash.in_bash
-def vertify_delta_lease(vg_uuid, host_id):
-    return bash.bash_r("sanlock direct read_leader -s lvm_%s:%s:/dev/mapper/%s-lvmlock:0" % (vg_uuid, host_id, vg_uuid))
+def read_delta_lease(vg_uuid, host_id):
+    return bash.bash_roe("sanlock direct read_leader -s lvm_%s:%s:/dev/mapper/%s-lvmlock:0" % (vg_uuid, host_id, vg_uuid))
```

Pylint (3.3.7)

[convention] Line too long (122/100)

(C0301)